python-django/README.md

```diff
@@ -39,6 +39,30 @@ Go to https://app.rookout.com and start debugging :)
    rook.start()
 ```
 
-[Python + Rookout]: https://docs.rookout.com/docs/sdk-setup.html
-[here]: https://github.com/GoogleCloudPlatform/nodejs-docs-samples/tree/master/appengine/hello-world
+## Usage
+
+- Log in to the [Rookout IDE](https://app.rookout.com/)
+- Set up your debug session by selecting the Python app you started. See [Debug session setup](https://docs.rookout.com/docs/debug-session-setup) for more information
+- Add the source code according to the following [instructions](https://docs.rookout.com/docs/source-repos/). In this case, use the [local filesystem option](https://docs.rookout.com/docs/source-repos/) to associate the code in your local 'python-django' folder.
+- Open the file 'manage.py'
```

Comment:
@jhendrick Since this is python and the instructions here are to clone the repo and run the app, the SDK is supposed to detect the remote origin and commit automatically and load the source files.